lib/semantic.d.ts

```diff
@@ -1,6 +1,6 @@
 import type { SemVer } from "semver";
 
+export function inc(input: string | SemVer): string | null;
 export function major(input: string | SemVer): number;
 export function minor(input: string | SemVer): number;
 export function patch(input: string | SemVer): number;
```

Comment:
```suggestion
export function inc(input: string): string | null;
export function major(input: string): number;
export function minor(input: string): number;
export function patch(input: string): number;
```

It seems we don't support `SemVer` class in js code, so remove it.